pkg/commands/workflows/plan_status_comments.go

```diff
@@ -66,16 +56,16 @@ Usage: {{ COMMAND }} [options]
 func (c *PlanStatusCommentCommand) Flags() *cli.FlagSet {
 	set := c.NewFlagSet()
 
-	c.GitHubFlags.Register(set)
-	c.RetryFlags.Register(set)
+	c.githubConfig.Register(set)
 
 	f := set.NewSection("COMMAND OPTIONS")
 
-	f.IntVar(&cli.IntVar{
-		Name:    "pull-request-number",
-		Target:  &c.flagPullRequestNumber,
-		Example: "100",
-		Usage:   "The GitHub pull request number to remove plan comments from.",
+	f.StringVar(&cli.StringVar{
+		Name:    "reporter",
+		Target:  &c.flagReporter,
+		Example: "github",
+		Default: "local",
+		Usage:   "The reporting strategy for Guardian status updates.",
```

Comment:
MIght be helpful to enumerate all the possible values if this flag in the Usage string. It looks like there are probably just two, but it's hard to say for sure.

Reply:
done